### DISCUSSION TITLE
where is the htdocs file after instal apache 2

### DISCUSSION THREAD

**Discussion by Jumbo125** [termux/termux-app#3286] [Q&A] [answered]
Hello 
i installed apache on termux and know i want to open the files  in the htdocs directory to edit the html file. but i want to open it with a editor or file explorer, not via termux command line.
Does anybody has a idea?

thank you

**Comment by sylirre** [comment, marked as answer]
One of variants is to change apache configuration to move htdocs to Termux home directory. Then you should be able to use any text editor that is able to access volumes over SAF, for example QuickEdit app.

Moving to shared storage is not recommended.

https://wiki.termux.com/wiki/Internal_and_external_storage
